nautobot/core/settings.py

```diff
@@ -117,6 +117,15 @@
 # integration tests if explicitly passed in with `nautobot-server test --tag integration`.
 TEST_RUNNER = "nautobot.core.tests.runner.NautobotTestRunner"
 
+#
+# django-slowtests
+#
+
+# django-slowtests setting and it is set so that only adding `--report` at the end of the `invoke` command
+# will output the performance test results. e.g. `invoke unittest --report`.
+TEST_ALWAYS_GENERATE_SLOW_REPORT = False
+TEST_PERFORMANCE_BASELINE_FILE = "nautobot/core/tests/performance_baselines.yml"
```

Comment:
These should be documented somewhere - normally these'd belong in `docs/configuration/optional-settings.md`, but since these are dev/test specific maybe `docs/development/testing.md` would be more appropriate.

Reply:
I would prefer now that we have sections for "Administratively Configurable Settings" and "Django Configuration Settings", that we just add these as optional settings, and then have clear references to them in the testing docs.

This is s similar pattern to how we have it in our installation docs, where the settings are explained in the context they are needed, but documented centrally in the configuration required/optional settings section.